test/http-server-v1.js

```diff
@@ -94,19 +109,19 @@ describe('v1 API', function() {
     const endpoint = '/v1/token/default';
 
     it('accepts GET requests', function() {
-      return util.acceptRequest(endpoint, 'GET', {}, {});
+      return util.acceptRequest(endpoint, 'GET', {}, requiredHeaders);
```

Comment:
Do you need the `requiredHeaders` for a GET request or can it be an empty object?